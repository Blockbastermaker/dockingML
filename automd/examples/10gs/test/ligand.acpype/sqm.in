Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=-1,
 /
   7     N       15.0880       10.7980       23.5470 
   6    CA       15.0100        9.9870       24.7920 
   6     C       16.1150        8.9240       24.8300 
   8     O       16.5200        8.5150       25.9400 
   6    CB       13.6350        9.3270       24.9080 
   6    CG       13.3940        8.7080       26.2710 
   6    CD       12.0450        8.0460       26.4020 
   8   OE1       11.2930        7.9360       25.4350 
   8   OXT       16.5780        8.5240       23.7440 
   7    N1       11.7260        7.6420       27.6280 
   6    C1       10.4720        6.9670       27.9340 
   6    C5       10.7260        5.4840       28.2060 
  16    SG       11.2910        4.5240       26.8100 
   6    C7        9.7290        3.8040       26.2620 
   6    CE        8.9300        3.1710       27.3700 
   6   CZ1        7.6400        3.6140       27.6500 
   6   CZ2        9.4640        2.1350       28.1330 
   6   CT1        6.8930        3.0370       28.6730 
   6   CT2        8.7230        1.5500       29.1610 
   6    CH        7.4370        2.0010       29.4300 
   6    C3        9.8340        7.5500       29.1800 
   8    O1       10.5220        8.0230       30.0840 
   7    N2        8.5120        7.4680       29.2290 
   6    C2        7.7400        7.9330       30.3660 
   6    C6        6.5550        7.0620       30.6330 
   6   CG1        5.3300        7.3150       30.0270 
   6   CG2        6.6830        5.9410       31.4410 
   6   CD1        4.2500        6.4590       30.2200 
   6   CD2        5.6110        5.0810       31.6400 
   6    C8        4.3920        5.3390       31.0270 
   6    C4        7.4520        9.4330       30.3540 
   8    O2        7.1160        9.9570       31.4330 
   8    O3        7.5690       10.0680       29.2840 
   1    H1       14.3520       11.4870       23.5480 
   1    H2       14.9820       10.1950       22.7460 
   1    H3       15.9820       11.2610       23.5030 
   1    H4       15.1480       10.6590       25.6520 
   1    H5       13.5580        8.5380       24.1450 
   1    H6       12.8630       10.0890       24.7270 
   1    H7       13.4680        9.5000       27.0310 
   1    H8       14.1720        7.9520       26.4520 
   1    H9       12.3740        7.8090       28.3710 
   1   H10        9.7860        7.0780       27.0810 
   1   H11        9.7850        5.0400       28.5620 
   1   H12       11.4880        5.4110       28.9960 
   1   H13        9.9480        3.0330       25.5080 
   1   H14        9.1210        4.6000       25.8070 
   1   H15        7.2120        4.4190       27.0640 
   1   H16       10.4670        1.7790       27.9260 
   1   H17        5.8900        3.3930       28.8800 
   1   H18        9.1500        0.7450       29.7480 
   1   H19        6.8580        1.5480       30.2270 
   1   H20        8.0270        7.0690       28.4510 
   1   H21        8.3980        7.7870       31.2360 
   1   H22        5.2150        8.1890       29.3970 
   1   H23        7.6320        5.7340       31.9230 
   1   H24        3.3000        6.6670       29.7410 
   1   H25        5.7260        4.2090       32.2730 
   1   H26        3.5540        4.6680       31.1780 

