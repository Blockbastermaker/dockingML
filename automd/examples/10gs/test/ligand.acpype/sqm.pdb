ATOM      1  N   VWW     1      11.884  10.766  27.043  1.00  0.00           N
ATOM      2  CA  VWW     1      12.512   9.539  26.476  1.00  0.00           C
ATOM      3  C   VWW     1      13.882   9.900  25.766  1.00  0.00           C
ATOM      4  O   VWW     1      14.669   8.945  25.585  1.00  0.00           O
ATOM      5  CB  VWW     1      11.590   8.835  25.480  1.00  0.00           C
ATOM      6  CG  VWW     1      12.037   7.409  25.221  1.00  0.00           C
ATOM      7  CD  VWW     1      11.242   6.430  26.064  1.00  0.00           C
ATOM      8  OE1 VWW     1      10.232   5.846  25.612  1.00  0.00           O
ATOM      9  OXT VWW     1      14.052  11.111  25.461  1.00  0.00           O
ATOM     10  N1  VWW     1      11.662   6.199  27.360  1.00  0.00           N
ATOM     11  C1  VWW     1      10.883   5.441  28.303  1.00  0.00           C
ATOM     12  C5  VWW     1      11.636   4.295  28.974  1.00  0.00           C
ATOM     13  SG  VWW     1      12.364   3.081  27.921  1.00  0.00           S
ATOM     14  C7  VWW     1      11.225   2.750  26.604  1.00  0.00           C
ATOM     15  CE  VWW     1       9.914   2.193  27.001  1.00  0.00           C
ATOM     16  CZ1 VWW     1       8.741   2.908  26.722  1.00  0.00           C
ATOM     17  CZ2 VWW     1       9.823   0.944  27.628  1.00  0.00           C
ATOM     18  CT1 VWW     1       7.499   2.381  27.073  1.00  0.00           C
ATOM     19  CT2 VWW     1       8.580   0.421  27.976  1.00  0.00           C
ATOM     20  CH  VWW     1       7.416   1.138  27.700  1.00  0.00           C
ATOM     21  C3  VWW     1      10.351   6.385  29.425  1.00  0.00           C
ATOM     22  O1  VWW     1      10.774   6.279  30.593  1.00  0.00           O
ATOM     23  N2  VWW     1       9.383   7.287  29.048  1.00  0.00           N
ATOM     24  C2  VWW     1       9.173   8.526  29.780  1.00  0.00           C
ATOM     25  C6  VWW     1       7.721   8.728  30.128  1.00  0.00           C
ATOM     26  CG1 VWW     1       6.796   9.087  29.140  1.00  0.00           C
ATOM     27  CG2 VWW     1       7.291   8.554  31.447  1.00  0.00           C
ATOM     28  CD1 VWW     1       5.455   9.272  29.473  1.00  0.00           C
ATOM     29  CD2 VWW     1       5.949   8.744  31.776  1.00  0.00           C
ATOM     30  C8  VWW     1       5.029   9.100  30.791  1.00  0.00           C
ATOM     31  C4  VWW     1       9.706   9.716  28.934  1.00  0.00           C
ATOM     32  O2  VWW     1      10.546  10.506  29.450  1.00  0.00           O
ATOM     33  O3  VWW     1       9.358   9.815  27.714  1.00  0.00           O
ATOM     34  H1  VWW     1      10.859  10.765  26.890  1.00  0.00           H
ATOM     35  H2  VWW     1      12.323  11.581  26.596  1.00  0.00           H
ATOM     36  H3  VWW     1      12.021  10.803  28.063  1.00  0.00           H
ATOM     37  H4  VWW     1      12.740   8.850  27.335  1.00  0.00           H
ATOM     38  H5  VWW     1      11.614   9.401  24.511  1.00  0.00           H
ATOM     39  H6  VWW     1      10.526   8.840  25.852  1.00  0.00           H
ATOM     40  H7  VWW     1      13.145   7.311  25.394  1.00  0.00           H
ATOM     41  H8  VWW     1      11.836   7.147  24.148  1.00  0.00           H
ATOM     42  H9  VWW     1      12.504   6.612  27.684  1.00  0.00           H
ATOM     43  H10 VWW     1       9.991   4.999  27.756  1.00  0.00           H
ATOM     44  H11 VWW     1      10.924   3.807  29.686  1.00  0.00           H
ATOM     45  H12 VWW     1      12.496   4.689  29.576  1.00  0.00           H
ATOM     46  H13 VWW     1      11.784   2.008  25.975  1.00  0.00           H
ATOM     47  H14 VWW     1      11.077   3.681  25.994  1.00  0.00           H
ATOM     48  H15 VWW     1       8.806   3.890  26.219  1.00  0.00           H
ATOM     49  H16 VWW     1      10.743   0.383  27.854  1.00  0.00           H
ATOM     50  H17 VWW     1       6.584   2.952  26.857  1.00  0.00           H
ATOM     51  H18 VWW     1       8.518  -0.559  28.471  1.00  0.00           H
ATOM     52  H19 VWW     1       6.436   0.726  27.979  1.00  0.00           H
ATOM     53  H20 VWW     1       9.165   7.355  28.074  1.00  0.00           H
ATOM     54  H21 VWW     1       9.778   8.486  30.736  1.00  0.00           H
ATOM     55  H22 VWW     1       7.152   9.243  28.107  1.00  0.00           H
ATOM     56  H23 VWW     1       8.014   8.268  32.226  1.00  0.00           H
ATOM     57  H24 VWW     1       4.734   9.557  28.695  1.00  0.00           H
ATOM     58  H25 VWW     1       5.617   8.609  32.815  1.00  0.00           H
ATOM     59  H26 VWW     1       3.972   9.247  31.051  1.00  0.00           H
