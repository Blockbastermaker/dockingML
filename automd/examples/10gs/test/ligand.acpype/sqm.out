            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 88

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: S : M.J.S.DEWAR et al. INORG. CHEM., 29, 3881, (1990)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      158
| QMMM:    diag iterations used for timing =        5
| QMMM:
| QMMM:              Internal diag routine = 0.042807 seconds
| QMMM:                 Dspev diag routine = 0.040571 seconds
| QMMM:                Dspevd diag routine = 0.025866 seconds
| QMMM:                Dspevx diag routine = 0.108159 seconds
| QMMM:                 Dsyev diag routine = 0.043834 seconds
| QMMM:                Dsyevd diag routine = 0.029467 seconds
| QMMM:                Dsyevr diag routine = 0.030264 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.017563 seconds
| QMMM:
| QMMM: Using dspevd routine (diag_routine=3).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N       15.0880   10.7980   23.5470
  QMMM:     2        2      C       15.0100    9.9870   24.7920
  QMMM:     3        3      C       16.1150    8.9240   24.8300
  QMMM:     4        4      O       16.5200    8.5150   25.9400
  QMMM:     5        5      C       13.6350    9.3270   24.9080
  QMMM:     6        6      C       13.3940    8.7080   26.2710
  QMMM:     7        7      C       12.0450    8.0460   26.4020
  QMMM:     8        8      O       11.2930    7.9360   25.4350
  QMMM:     9        9      O       16.5780    8.5240   23.7440
  QMMM:    10       10      N       11.7260    7.6420   27.6280
  QMMM:    11       11      C       10.4720    6.9670   27.9340
  QMMM:    12       12      C       10.7260    5.4840   28.2060
  QMMM:    13       13      S       11.2910    4.5240   26.8100
  QMMM:    14       14      C        9.7290    3.8040   26.2620
  QMMM:    15       15      C        8.9300    3.1710   27.3700
  QMMM:    16       16      C        7.6400    3.6140   27.6500
  QMMM:    17       17      C        9.4640    2.1350   28.1330
  QMMM:    18       18      C        6.8930    3.0370   28.6730
  QMMM:    19       19      C        8.7230    1.5500   29.1610
  QMMM:    20       20      C        7.4370    2.0010   29.4300
  QMMM:    21       21      C        9.8340    7.5500   29.1800
  QMMM:    22       22      O       10.5220    8.0230   30.0840
  QMMM:    23       23      N        8.5120    7.4680   29.2290
  QMMM:    24       24      C        7.7400    7.9330   30.3660
  QMMM:    25       25      C        6.5550    7.0620   30.6330
  QMMM:    26       26      C        5.3300    7.3150   30.0270
  QMMM:    27       27      C        6.6830    5.9410   31.4410
  QMMM:    28       28      C        4.2500    6.4590   30.2200
  QMMM:    29       29      C        5.6110    5.0810   31.6400
  QMMM:    30       30      C        4.3920    5.3390   31.0270
  QMMM:    31       31      C        7.4520    9.4330   30.3540
  QMMM:    32       32      O        7.1160    9.9570   31.4330
  QMMM:    33       33      O        7.5690   10.0680   29.2840
  QMMM:    34       34      H       14.3520   11.4870   23.5480
  QMMM:    35       35      H       14.9820   10.1950   22.7460
  QMMM:    36       36      H       15.9820   11.2610   23.5030
  QMMM:    37       37      H       15.1480   10.6590   25.6520
  QMMM:    38       38      H       13.5580    8.5380   24.1450
  QMMM:    39       39      H       12.8630   10.0890   24.7270
  QMMM:    40       40      H       13.4680    9.5000   27.0310
  QMMM:    41       41      H       14.1720    7.9520   26.4520
  QMMM:    42       42      H       12.3740    7.8090   28.3710
  QMMM:    43       43      H        9.7860    7.0780   27.0810
  QMMM:    44       44      H        9.7850    5.0400   28.5620
  QMMM:    45       45      H       11.4880    5.4110   28.9960
  QMMM:    46       46      H        9.9480    3.0330   25.5080
  QMMM:    47       47      H        9.1210    4.6000   25.8070
  QMMM:    48       48      H        7.2120    4.4190   27.0640
  QMMM:    49       49      H       10.4670    1.7790   27.9260
  QMMM:    50       50      H        5.8900    3.3930   28.8800
  QMMM:    51       51      H        9.1500    0.7450   29.7480
  QMMM:    52       52      H        6.8580    1.5480   30.2270
  QMMM:    53       53      H        8.0270    7.0690   28.4510
  QMMM:    54       54      H        8.3980    7.7870   31.2360
  QMMM:    55       55      H        5.2150    8.1890   29.3970
  QMMM:    56       56      H        7.6320    5.7340   31.9230
  QMMM:    57       57      H        3.3000    6.6670   29.7410
  QMMM:    58       58      H        5.7260    4.2090   32.2730
  QMMM:    59       59      H        3.5540    4.6680   31.1780

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -206.5663 kcal/mol        0.7493 kcal/(mol*A)
xmin    20     -208.2470 kcal/mol        1.7961 kcal/(mol*A)
xmin    30     -209.7391 kcal/mol        1.0307 kcal/(mol*A)
xmin    40     -210.8213 kcal/mol        1.2040 kcal/(mol*A)
xmin    50     -211.4386 kcal/mol        0.6961 kcal/(mol*A)
xmin    60     -211.9478 kcal/mol        0.3175 kcal/(mol*A)
xmin    70     -212.9877 kcal/mol        0.7580 kcal/(mol*A)
xmin    80     -218.2178 kcal/mol        3.1724 kcal/(mol*A)
xmin    90     -234.4690 kcal/mol        2.1023 kcal/(mol*A)
xmin   100     -241.6274 kcal/mol        0.5788 kcal/(mol*A)
xmin   110     -243.7531 kcal/mol        0.2795 kcal/(mol*A)
xmin   120     -244.1950 kcal/mol        0.1575 kcal/(mol*A)
xmin   130     -244.5902 kcal/mol        0.3091 kcal/(mol*A)
xmin   140     -245.6307 kcal/mol        1.8582 kcal/(mol*A)
xmin   150     -247.8166 kcal/mol        0.2098 kcal/(mol*A)
xmin   160     -247.8656 kcal/mol        0.1328 kcal/(mol*A)
xmin   170     -248.0017 kcal/mol        0.0619 kcal/(mol*A)
xmin   180     -248.0205 kcal/mol        0.0296 kcal/(mol*A)
xmin   190     -248.0241 kcal/mol        0.0212 kcal/(mol*A)
xmin   200     -248.0245 kcal/mol        0.0042 kcal/(mol*A)
xmin   210     -248.0247 kcal/mol        0.0018 kcal/(mol*A)
xmin   220     -248.0247 kcal/mol        0.0010 kcal/(mol*A)
xmin   230     -248.0247 kcal/mol        0.0006 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -38.4844  -37.7947  -37.5227  -37.0296  -36.4836  -35.2702  -33.9057  -33.6080
 -32.8037  -32.0758  -31.8301  -31.2682  -31.0999  -29.7419  -29.6399  -29.1486
 -29.1010  -26.0505  -25.4503  -25.1066  -22.6714  -21.5607  -21.2408  -20.9930
 -20.5183  -20.1022  -19.3523  -18.9218  -17.6639  -17.5214  -16.5425  -16.1355
 -16.0098  -15.8036  -15.6260  -15.3952  -14.9896  -14.6515  -14.5644  -14.3797
 -14.3236  -14.2829  -13.9509  -13.6545  -13.4286  -13.0982  -12.8051  -12.6780
 -12.6110  -12.5304  -12.4233  -12.3045  -12.1996  -12.1049  -11.9820  -11.9147
 -11.8690  -11.8036  -11.7460  -11.1478  -11.1030  -11.0159  -10.7471  -10.6596
 -10.5067  -10.4716  -10.3886  -10.2242  -10.1004   -9.7341   -9.6531   -9.4956
  -9.1918   -9.0188   -8.6856   -8.3083   -7.9934   -7.8212   -7.6923   -7.4318
  -7.3868   -7.2652   -7.1982   -7.1487   -6.8333   -6.6236   -6.5474   -6.3974
   1.9381    2.2400    2.6796    2.8183    2.8459    3.2771    3.5118    3.5546
   4.0876    4.3392    4.5772    4.6558    5.2424    5.4399    5.4744    5.5074
   5.5515    5.5872    5.6817    5.7181    5.7788    5.8287    5.8915    5.9254
   5.9656    5.9890    6.1539    6.1926    6.2006    6.2448    6.2545    6.3122
   6.3244    6.3585    6.3951    6.5069    6.5566    6.6072    6.6152    6.6505
   6.7488    6.7828    6.8599    6.8701    6.9016    6.9288    7.0454    7.1261
   7.1702    7.2315    7.2465    7.2983    7.3605    7.3675    7.4241    7.4970
   7.7884    7.7959    7.8378    7.8941    8.1175    8.3001    8.4108    8.7644
   9.0727    9.3667   10.1012   10.2868   10.9968   11.4395

 Heat of formation   =       -248.02472394 kcal/mol  (      -10.75515910 eV)

 Total SCF energy    =    -139372.23555921 kcal/mol  (    -6043.63364812 eV)
 Electronic energy   =   -1196429.24447062 kcal/mol  (   -51881.06519538 eV)
 Core-core repulsion =    1057057.00891142 kcal/mol  (    45837.43154726 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      N                 -0.073
     2      C                 -0.183
     3      C                  0.341
     4      O                 -0.509
     5      C                 -0.160
     6      C                 -0.182
     7      C                  0.312
     8      O                 -0.410
     9      O                 -0.552
    10      N                 -0.375
    11      C                 -0.000
    12      C                 -0.274
    13      S                  0.050
    14      C                 -0.242
    15      C                 -0.076
    16      C                 -0.123
    17      C                 -0.119
    18      C                 -0.131
    19      C                 -0.139
    20      C                 -0.134
    21      C                  0.275
    22      O                 -0.378
    23      N                 -0.328
    24      C                 -0.006
    25      C                 -0.053
    26      C                 -0.106
    27      C                 -0.115
    28      C                 -0.145
    29      C                 -0.148
    30      C                 -0.141
    31      C                  0.316
    32      O                 -0.569
    33      O                 -0.603
    34      H                  0.271
    35      H                  0.242
    36      H                  0.262
    37      H                  0.115
    38      H                  0.099
    39      H                  0.119
    40      H                  0.158
    41      H                  0.113
    42      H                  0.243
    43      H                  0.142
    44      H                  0.122
    45      H                  0.138
    46      H                  0.122
    47      H                  0.136
    48      H                  0.164
    49      H                  0.136
    50      H                  0.132
    51      H                  0.123
    52      H                  0.123
    53      H                  0.233
    54      H                  0.134
    55      H                  0.169
    56      H                  0.134
    57      H                  0.119
    58      H                  0.117
    59      H                  0.114
 Total Mulliken Charge =      -1.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE   -58.024  -41.899 -132.327  150.442
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N       11.8844   10.7657   27.0429
  QMMM:     2        2      C       12.5123    9.5391   26.4758
  QMMM:     3        3      C       13.8816    9.9003   25.7663
  QMMM:     4        4      O       14.6694    8.9451   25.5852
  QMMM:     5        5      C       11.5896    8.8349   25.4803
  QMMM:     6        6      C       12.0366    7.4086   25.2206
  QMMM:     7        7      C       11.2422    6.4296   26.0642
  QMMM:     8        8      O       10.2316    5.8458   25.6122
  QMMM:     9        9      O       14.0517   11.1106   25.4606
  QMMM:    10       10      N       11.6616    6.1993   27.3596
  QMMM:    11       11      C       10.8832    5.4413   28.3034
  QMMM:    12       12      C       11.6358    4.2949   28.9740
  QMMM:    13       13      S       12.3635    3.0812   27.9209
  QMMM:    14       14      C       11.2253    2.7502   26.6043
  QMMM:    15       15      C        9.9135    2.1929   27.0011
  QMMM:    16       16      C        8.7407    2.9084   26.7219
  QMMM:    17       17      C        9.8233    0.9436   27.6279
  QMMM:    18       18      C        7.4989    2.3813   27.0733
  QMMM:    19       19      C        8.5798    0.4206   27.9757
  QMMM:    20       20      C        7.4159    1.1384   27.7001
  QMMM:    21       21      C       10.3515    6.3848   29.4248
  QMMM:    22       22      O       10.7739    6.2790   30.5932
  QMMM:    23       23      N        9.3833    7.2866   29.0477
  QMMM:    24       24      C        9.1730    8.5260   29.7804
  QMMM:    25       25      C        7.7212    8.7278   30.1275
  QMMM:    26       26      C        6.7962    9.0873   29.1405
  QMMM:    27       27      C        7.2906    8.5544   31.4470
  QMMM:    28       28      C        5.4553    9.2722   29.4734
  QMMM:    29       29      C        5.9487    8.7437   31.7763
  QMMM:    30       30      C        5.0293    9.1002   30.7905
  QMMM:    31       31      C        9.7056    9.7159   28.9336
  QMMM:    32       32      O       10.5461   10.5064   29.4498
  QMMM:    33       33      O        9.3577    9.8145   27.7141
  QMMM:    34       34      H       10.8593   10.7653   26.8903
  QMMM:    35       35      H       12.3228   11.5814   26.5963
  QMMM:    36       36      H       12.0215   10.8025   28.0628
  QMMM:    37       37      H       12.7404    8.8496   27.3353
  QMMM:    38       38      H       11.6138    9.4006   24.5110
  QMMM:    39       39      H       10.5260    8.8397   25.8523
  QMMM:    40       40      H       13.1452    7.3113   25.3943
  QMMM:    41       41      H       11.8362    7.1469   24.1476
  QMMM:    42       42      H       12.5043    6.6122   27.6839
  QMMM:    43       43      H        9.9909    4.9990   27.7562
  QMMM:    44       44      H       10.9243    3.8069   29.6864
  QMMM:    45       45      H       12.4956    4.6886   29.5764
  QMMM:    46       46      H       11.7835    2.0076   25.9751
  QMMM:    47       47      H       11.0773    3.6810   25.9945
  QMMM:    48       48      H        8.8057    3.8903   26.2194
  QMMM:    49       49      H       10.7428    0.3830   27.8543
  QMMM:    50       50      H        6.5839    2.9519   26.8570
  QMMM:    51       51      H        8.5175   -0.5587   28.4712
  QMMM:    52       52      H        6.4358    0.7263   27.9790
  QMMM:    53       53      H        9.1648    7.3549   28.0744
  QMMM:    54       54      H        9.7785    8.4858   30.7361
  QMMM:    55       55      H        7.1520    9.2431   28.1067
  QMMM:    56       56      H        8.0136    8.2685   32.2257
  QMMM:    57       57      H        4.7336    9.5574   28.6950
  QMMM:    58       58      H        5.6165    8.6094   32.8152
  QMMM:    59       59      H        3.9721    9.2471   31.0513

           --------- Calculation Completed ----------

